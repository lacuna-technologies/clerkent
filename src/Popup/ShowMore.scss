@import "../styles/core.scss";

#show-more {
  width: 100%; 
  text-align: center; 
  border-bottom: 1px solid $border-color; 
  line-height: 0.1em;
  margin: 10px 0 20px;
  cursor: pointer;

  > span {
    background: white;
    padding: 0 10px;
  }
}