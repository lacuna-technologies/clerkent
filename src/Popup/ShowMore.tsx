import React from 'react'

const ShowMore = ({ onClick }) => {
  return (
    <div className="w-full text-center border-b border-solid border-gray-400 leading-[0.1rem] mt-2 mb-4 cursor-pointer" onClick={onClick}>
        <span className="bg-white px-2">
          <small>Show More</small>
        </span>
    </div>
  )
}

export default ShowMore