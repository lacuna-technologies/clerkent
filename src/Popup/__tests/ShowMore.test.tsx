import React from 'react'
import renderer from 'react-test-renderer'
import ShowMore from '../ShowMore'

describe(`ShowMore`, () => {
  it(`renders without error`, () => {
    const tree = renderer.create(
      <ShowMore
        onClick={() => {}}
      />,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})