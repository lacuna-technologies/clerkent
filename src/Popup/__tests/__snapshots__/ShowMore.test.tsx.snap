// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowMore renders without error 1`] = `
<div
  className="w-full text-center border-b border-solid border-gray-400 leading-[0.1rem] mt-2 mb-4 cursor-pointer"
  onClick={[Function]}
>
  <span
    className="bg-white px-2"
  >
    <small>
      Show More
    </small>
  </span>
</div>
`;
